### DISCUSSION TITLE
Lock Tool Windows like Inspector

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#6065] [Editor]
Lock tool windows like the Insprector. In Unity you can lock a view so you can click everywhere you want without loosing the current view. A use case could be, keep shader editor open while clicking on another object to e.g. move it. Or click on camera while editing env post fx. You can click again on the lock icon to unlock and show the current context (normal behavior).

![Unity](https://user-images.githubusercontent.com/53160891/211200620-2a97c89b-b691-473b-88c0-c11c51611a09.png)
<sup>Random screenshot from web. Not actual latest Unity version. Just for illustration. See the lock icon.</sup>

**Comment by amarraff** [comment]
I'd also like to see this feature. Just now, I was trying to drag 40 nodes into an exported array; I keep clicking away from the node that contains the array when I select them.

**Comment by yosimba2000** [comment]
I would like this feature as well, same exact usecase as Amarraff

**Reply by Calinou** [reply]
@yosimba2000 Please don't bump issues without contributing significant new information. Use the :+1: reaction button on the first post instead.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/9265.

**Comment by theraot** [comment]
Cross referencing: https://github.com/godotengine/godot-proposals/discussions/11900
